scripts/clean-local-docker.sh

```diff
@@ -8,7 +8,7 @@ function usage() {
 
 [ $# -ne 1 ] && (usage; exit 1)
 
-TMPDIR=/tmp/parachain_dev
+TMPDIR=${TMPDIR:-"/tmp/parachain_dev"}
```

Comment:
The default scares me tbh. Can we just fail if it's not set?

Reply:
I've updated this to: 
```
TMPDIR=${TMPDIR:-"/tmp/parachain_dev*"}
``` 
Since we are creating a new directory every time, If we don't specify the exact directory we want to clean-up, it will clean up all with this pattern. It will however not delete files that were created by another user since we don't have permissions for it. I don't see a scenario where this could be problematic, Let me know what do you think.